cmd/root_test.go

```diff
@@ -378,6 +392,10 @@ func TestNewCommandWithErrors(t *testing.T) {
 			desc: "enabling a Prometheus port without a namespace",
 			args: []string{"--htto-port", "1111", "proj:region:inst"},
 		},
+		{
+			desc: "using an invalid url for api-endpoint-url",
+			args: []string{"--api-endpoint-url", "https://user:abc{DEf1=ghi@example.com:5432/db?sslmode=require", "proj:region:inst"},
```

Comment:
This url will throw an error from golang url.Parse() because the user:password part is not properly escaped.